recipes/cymysql/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "cymysql" %}
+{% set version = "0.9.14" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2a20c6d5373a16590b75f149a7cb2215a6d3bb2fc856b1b75f437ccf1e716d4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
```

I think you will need to add the compilers here and remove ``noarch`` as well
```suggestion
requirements:
  build:
    - {{ compiler('cxx') }}
    - {{ compiler('c') }}
```